app/javascript/listings/listingDashboard.jsx

```diff
@@ -1,232 +1,211 @@
-import { h, Component } from 'preact';
+import { h } from 'preact';
+import { useState, useEffect } from 'preact/hooks';
 import { ListingRow } from './dashboard/listingRow';
 
-export class ListingDashboard extends Component {
-  state = {
-    listings: [],
-    orgListings: [],
-    orgs: [],
-    userCredits: 0,
-    selectedListings: 'user',
-    filter: 'All',
-    sort: 'created_at',
-  };
+export const ListingDashboard = () => {
+  const [listings, setListings] = useState([]);
+  const [orgListings, setOrgListings] = useState([]);
+  const [orgs, setOrgs] = useState([]);
+  const [userCredits, setUserCredits] = useState(0);
+  const [selectedListings, setSelectedListings] = useState('user');
+  const [filter, setFilter] = useState('');
+  const [sort, setSort] = useState('created_at');
```

Comment:
(not blocking) This state could be a good candidate for `useReducer` as we have so many `useState`